classes/order/Order.php

```diff
@@ -1816,6 +1816,16 @@ public function getOrderPaymentCollection()
         return $order_payments;
     }
 
+    /**
+     * Indicates if order has any associated payments.
+     *
+     * @return bool
+     */
+    public function hasPayments(): bool
+    {
+        return $this->getOrderPaymentCollection()->count() > 0;
```

Comment:
```suggestion
        return $this->getOrderPaymentCollection()->count();
```

Reply:
this modification will not return a boolean anymore 🤔